03.asynchronous/callback.js

```diff
@@ -0,0 +1,45 @@
+import sqlite3 from "sqlite3";
+import timers from "timers/promises";
+
+const db = new sqlite3.Database(":memory:");
+
+// 1. コールバック（エラーなし）
+db.run(
+  "CREATE TABLE books (id INTEGER PRIMARY KEY AUTOINCREMENT, title TEXT NOT NULL UNIQUE)",
+  () => {
+    db.run(
+      "INSERT INTO books(title) VALUES('ゼロからわかるRuby超入門')",
```

Comment:
`CREATE TABLE` とスタイルを揃えるなら以下のようにしたほうがよいと思います。

```suggestion
      "INSERT INTO books (title) VALUES ('ゼロからわかるRuby超入門')",
```

Reply:
こちらも他のSQL文と同じスタイルになるよう空行を挿入してコミットしました🙇‍♂️